crates/pathfinder/src/bin/pathfinder/main.rs

```diff
@@ -171,6 +171,14 @@ Hint: This is usually caused by exceeding the file descriptor limit of your syst
     .await
     .context("Verifying database")?;
 
+    sync_storage
```

Comment:
We should log that this is happening in case it takes a long time. However, users might freak out if they see
```
INFO Pruning state tries
```
espectially if they have an archive node. So maybe
```
tracing::info!("Cleaning up state trie");
```

Reply:
Or just do the logging where it's actually happening in `prune_tries()` so that it never gets printed unless you've enabled pruning?

Reply:
Figured that inside might be too noisy 🤷 `INFO` level seems a bit high for during sync, but guess its okay.